e2e/pages/respondToClaim/confirmDetails.page.js

```diff
@@ -0,0 +1,17 @@
+const {I} = inject();
+
+module.exports = {
+
+  fields: {
+    solicitorReferences: {
+      id: '#solicitorReferences_defendantReference'
+    }
+  },
+
+  async confirmReference() {
+    I.waitForElement(this.fields.solicitorReferences.id);
+    I.fillField(this.fields.solicitorReferences.id, 'Updated Defendant Reference');
+
+    await I.clickContinue();
```

Comment:
can you not use the exisiting solicitor reference page?

Reply:
whether directly or as a separate fragment I am not sure which is cleaner

Reply:
done